roles/common/tasks/sysctl.yml

```diff
@@ -0,0 +1,81 @@
+# necessary to run a lot of containers, each which systemd launching several inotify
+- name: increase fs.inotify.max_user_instances on host
+  sysctl:
+    name: fs.inotify.max_user_instances
+    value: 1024
+    sysctl_file: /etc/sysctl.d/ansible.conf
```

Comment:
Est-ce qu'il ne faudrait pas garder le when: "not 'vm' in group_names" pour ne pas changer ces sysctl sur les containers ?

Reply:
oui, possible, je pensais que c'était ce que faisait ``when: "'proxmox' in group_names"``.

Si on ne veut appliquer que sur les hosts, il faut utiliser ``when: "not 'vm' in group_names`` ?

Reply:
J'avais raté le proxmox in group_names - c'est bon du coup.